### PR TITLE
Remove no longer free AWS service

### PR DESCRIPTION
**Pull request by thispsj**
AWS Glacier is no longer present in Always free or 12 mo free list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,6 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
     * [EBS](https://aws.amazon.com/ebs/) - 30GB per month of General Purpose (SSD) or Magnetic(12mo)
     * [Elastic Load Balancing](https://aws.amazon.com/elasticloadbalancing/) - 750 hours per month(12mo)
     * [RDS](https://aws.amazon.com/rds/) - 750 hours per month of db.t2.micro, db.t3.micro, or db.t4g.micro, 20GB of General Purpose (SSD) storage, 20GB of storage backups
-    * [Glacier](https://aws.amazon.com/glacier) - 10GB long-term object storage
```
Comment (ripienaar) [inline README.md:104]: Still listed here. Storage is S3 (with free tier) and first 10GB retrieved via glacier is free?

https://aws.amazon.com/s3/glacier/pricing/

Comment (thispsj) [reply README.md:104]: @ripienaar so sorry for that. I am feeling dumb now the way these guys have listed their free tier limits at so many different places.

Comment (ripienaar) [reply README.md:104]: Yup not your fault don’t worry :)

Comment (thispsj) [reply README.md:104]: I opened this PR because glacier was not listed [here](https://aws.amazon.com/free).

Comment (ripienaar) [reply README.md:104]: That’s weird. Maybe it is going away. Hard to say Amazon docs always a struggle.

Comment (thispsj) [reply README.md:104]: Yeah 😅

**Pull request closed without merging**